scripts/lint.sh

```diff
@@ -15,3 +13,5 @@ taplo format
 
 # Format sql files
 find migrations -type f -name '*.sql' -exec pg_format -i {} +
+
+cargo clippy --workspace --fix --allow-staged --allow-dirty --tests --all-targets --all-features -- -D warnings
```

Comment:
Run this last so you can simply do Ctrl+C if you only need formatting. So far I had to comment out the clippy line manually.